recipe/meta.yaml

```diff
@@ -79,7 +79,9 @@ outputs:
         - requests
         - future
         - six
-        - mkl-devel {{ mkl }}   # [x86]
+        - mkl==2021.4.0         # [x86]
+        - mkl-include==2021.4.0 # [x86]
+        - mkl-devel==2021.4.0   # [x86]- 
```

Comment:
Maybe these pins could be moved to `conda_build_config.yaml`. Then when resolving upstream changes would prefer upstream's changes (then could readd the `-devel` part). IIUC `-include` comes with `-devel`

https://github.com/conda-forge/pytorch-cpu-feedstock/blob/229b44730ae4e9b1cb144910f0a3a25cee0e7391/recipe/conda_build_config.yaml#L10-L13

Reply:
They're also really old (>2 years). Do we really need to pin mkl that tightly?

Reply:
Yeah am wondering the same. It also deviates from [the current global pinnings]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/023ae6b4004f90fb80116d7010ba5348be75757e/recipe/conda_build_config.yaml#L556-L559 )

Maybe this was needed to get the build working, in which case there probably needs to be some investigation into how to bump the versions (and fix whatever issue occurs when using more recent versions)

Reply:
See this comment for the rationale: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/172#issuecomment-1649225752